packages/editor/src/panels/hierarchy/hierarchynode.tsx

```diff
@@ -59,9 +59,10 @@ import React, { KeyboardEvent, useEffect, useRef } from 'react'
 import { useDrag } from 'react-dnd'
 import { getEmptyImage } from 'react-dnd-html5-backend'
 import { useTranslation } from 'react-i18next'
+import { HiOutlineLockClosed, HiOutlineLockOpen } from 'react-icons/hi'
```

Comment:
Lets use the IR design library icon of lock and unlock (sm sizing).
https://www.figma.com/design/ln2VDACenFEkjVeHkowxyi/iR-Engine-Design-Library-File?node-id=2180-8134&m=dev
![image](https://github.com/user-attachments/assets/54f48b0a-fa61-4b19-9456-c6f04e12871e)